nominees/upyog.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "UPYOG",
+  "aliases": [
+    "Urban Platform for Delivery of online Governance"
+  ],
+  "description": "The Centre for Digital Governance at the National Institute of Urban Affairs has launched the Urban Platform for deliverY of Online Governance programme as one of the flagship initiatives under the National Urban Digital Mission. The UPYOG program aims at supporting states in implementing integrated e-governance and digital delivery of municipal services to citizens",
+  "website": "https://upyog.niua.org/",
```

Comment:
This website requires login, which I'm assuming is the digital solution website where it is being deployed. We need some information website on the project itself.